schema.prisma

```diff
@@ -0,0 +1,25 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url = env("POSTGRES_PRISMA_URL")
+  directUrl = env("POSTGRES_URL_NON_POOLING")
+}
+
+enum SettingType {
+  ENABLED
+  DISABLED
+  WITHIN_WORKING_HOURS
+}
+
+model Setting {
+  id String @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+  type SettingType
+  workingHours Json?	@db.JsonB
+  senderId  String @db.Uuid @unique
+  message String?
+  createdAt DateTime @db.Timestamptz() @default(now()) @ignore
+  updatedAt DateTime @updatedAt @ignore
+}
```

Comment:
did you forget timezone here @sazanrjb

Reply:
timezone is inside workingHours
```
workingHours: {
  timezone: string,
  data: [
    { weekday: string, startTime: string, endTime: string}
  ]
}
```

let me know if it should be outside